### DISCUSSION TITLE
problem to activate the axtentions

### DISCUSSION THREAD

**Discussion by BS-eng2023** [SVG-Edit/svgedit#902] [General]
Dear all, 
I have the problem that I can't activate the axtentions 
![grafik](https://github.com/SVG-Edit/svgedit/assets/135705834/ed99640d-f707-45aa-a24d-49d0a7b0251f)
could anyone help me in this issue?
Best Regards

**Comment by jfhenon** [comment]
Hello. Could you give the content of the dev console?

**Reply by BS-eng2023** [reply]
Thanks for your Replay, do you mean this 
D:\SVG\quicknanobrowser_svgedit_prototype_source_gar_20230521\svgedit-7.2.0\svgedit-7.2.0\src>npm run build

> svgedit@7.2.0 build
> rollup -c

loaded rollup.config.js with warnings
(!) Unused external imports
default imported from external module "rollup-plugin-progress" but never used in "rollup.config.js"

src/editor/Editor.js → dist/editor/Editor.js, dist/editor/xdomain-Editor.js, dist/editor/iife-Editor.js...
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
recreating dist
[BABEL] Note: The code generator has deoptimised the styling of D:\SVG\quicknanobrowser_svgedit_prototype_source_gar_20230521\svgedit-7.2.0\svgedit-7.2.0\packages\svgcanvas\dist\svgcanvas.js as it exceeds the max of 500KB.
┌────────────────────────────────────────┐
│                                        │
│   Destination: dist/editor/Editor.js   │
│   Bundle Size:  2.17 MB                │
│   Minified Size:  2.04 MB              │
│   Gzipped Size:  464.42 KB             │
│                                        │
└────────────────────────────────────────┘
┌────────────────────────────────────────────────┐
│                                                │
│   Destination: dist/editor/xdomain-Editor.js   │
│   Bundle Size:  2.17 MB                        │
│   Minified Size:  2.04 MB                      │
│   Gzipped Size:  464.42 KB                     │
│                                                │
└────────────────────────────────────────────────┘
┌─────────────────────────────────────────────┐
│                                             │
│   Destination: dist/editor/iife-Editor.js   │
│   Bundle Size:  2.17 MB                     │
│   Minified Size:  2.01 MB                   │
│   Gzipped Size:  461.19 KB                  │
│                                             │
└─────────────────────────────────────────────┘
created dist/editor/Editor.js, dist/editor/xdomain-Editor.js, dist/editor/iife-Editor.js in 2m 36.6s

./src/editor/extensions/ext-eyedropper/ext-eyedropper.js → dist/editor/extensions/ext-eyedropper...
created dist/editor/extensions/ext-eyedropper in 647ms

./src/editor/extensions/ext-grid/ext-grid.js → dist/editor/extensions/ext-grid...
created dist/editor/extensions/ext-grid in 530ms

./src/editor/extensions/ext-helloworld/ext-helloworld.js → dist/editor/extensions/ext-helloworld...
created dist/editor/extensions/ext-helloworld in 466ms

./src/editor/extensions/ext-markers/ext-markers.js → dist/editor/extensions/ext-markers...
created dist/editor/extensions/ext-markers in 565ms

./src/editor/extensions/ext-opensave/ext-opensave.js → dist/editor/extensions/ext-opensave...
created dist/editor/extensions/ext-opensave in 784ms

./src/editor/extensions/ext-overview_window/ext-overview_window.js → dist/editor/extensions/ext-overview_window...
created dist/editor/extensions/ext-overview_window in 480ms

./src/editor/extensions/ext-panning/ext-panning.js → dist/editor/extensions/ext-panning...
created dist/editor/extensions/ext-panning in 440ms

./src/editor/extensions/ext-polystar/ext-polystar.js → dist/editor/extensions/ext-polystar...
created dist/editor/extensions/ext-polystar in 661ms

./src/editor/extensions/ext-shapes/ext-shapes.js → dist/editor/extensions/ext-shapes...
created dist/editor/extensions/ext-shapes in 530ms

./src/editor/extensions/ext-storage/ext-storage.js → dist/editor/extensions/ext-storage...
created dist/editor/extensions/ext-storage in 590ms
![grafik](https://github.com/SVG-Edit/svgedit/assets/135705834/5d3bfaf4-33e5-42af-8a26-451d25be8e81)

**Reply by BS-eng2023** [reply]
or this 
Browser-Konsolenmodus
Nur übergeordneter Prozess(Schnell)
Multiprozess(Langsamer)
PushService: receivedPushMessage: Error notifying app Error: No record for key ID 74732b2f-f06d-4619-8a67-b7281f423069
    _updateRecordAfterPush resource://gre/modules/PushService.sys.mjs:845
[PushService.sys.mjs:813](resource://gre/modules/PushService.sys.mjs)
Given tab is not restoring. [SessionStore.sys.mjs:6011:15](resource:///modules/sessionstore/SessionStore.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin http:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 394"  data: no]. [PurgeTrackerService.sys.mjs:398:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin https:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 411"  data: no]. [PurgeTrackerService.sys.mjs:415:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin http:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 394"  data: no]. [PurgeTrackerService.sys.mjs:398:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin https:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 411"  data: no]. [PurgeTrackerService.sys.mjs:415:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin http:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 394"  data: no]. [PurgeTrackerService.sys.mjs:398:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin https:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 411"  data: no]. [PurgeTrackerService.sys.mjs:415:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
Given tab is not restoring. [SessionStore.sys.mjs:6011:15](resource:///modules/sessionstore/SessionStore.sys.mjs)
NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIFaviconService.replaceFaviconDataFromDataURL]
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:226
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:729
    setIconFromLink resource:///actors/LinkHandlerParent.sys.mjs:139
    receiveMessage resource:///actors/LinkHandlerParent.sys.mjs:60
[LinkHandlerParent.sys.mjs:148:17](resource:///actors/LinkHandlerParent.sys.mjs)
PushService: receivedPushMessage: Error notifying app Error: No record for key ID 74732b2f-f06d-4619-8a67-b7281f423069
    _updateRecordAfterPush resource://gre/modules/PushService.sys.mjs:845
[PushService.sys.mjs:813](resource://gre/modules/PushService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin http:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 394"  data: no]. [PurgeTrackerService.sys.mjs:398:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin https:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 411"  data: no]. [PurgeTrackerService.sys.mjs:415:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin http:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 394"  data: no]. [PurgeTrackerService.sys.mjs:398:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin https:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 411"  data: no]. [PurgeTrackerService.sys.mjs:415:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin http:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 394"  data: no]. [PurgeTrackerService.sys.mjs:398:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
*** PurgeTrackerService:: Creating principal from origin https:// led to error [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIScriptSecurityManager.createContentPrincipalFromOrigin]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/PurgeTrackerService.sys.mjs :: purgeTrackingCookieJars :: line 411"  data: no]. [PurgeTrackerService.sys.mjs:415:21](resource://gre/modules/PurgeTrackerService.sys.mjs)
NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIFaviconService.replaceFaviconDataFromDataURL]
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:226
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:729
    setIconFromLink resource:///actors/LinkHandlerParent.sys.mjs:139
    receiveMessage resource:///actors/LinkHandlerParent.sys.mjs:60
[LinkHandlerParent.sys.mjs:148:17](resource:///actors/LinkHandlerParent.sys.mjs)
SearchSuggestionController found an unexpected string value: Unexpected response, searchString does not match remote response [SearchSuggestionController.sys.mjs:634:17](resource://gre/modules/SearchSuggestionController.sys.mjs)
NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIFaviconService.replaceFaviconDataFromDataURL]
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:226
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:729
    setIconFromLink resource:///actors/LinkHandlerParent.sys.mjs:139
    receiveMessage resource:///actors/LinkHandlerParent.sys.mjs:60
[LinkHandlerParent.sys.mjs:148:17](resource:///actors/LinkHandlerParent.sys.mjs)
PushService: receivedPushMessage: Error notifying app Error: No record for key ID 74732b2f-f06d-4619-8a67-b7281f423069
    _updateRecordAfterPush resource://gre/modules/PushService.sys.mjs:845
[PushService.sys.mjs:813](resource://gre/modules/PushService.sys.mjs)
PushService: receivedPushMessage: Error notifying app Error: No record for key ID 74732b2f-f06d-4619-8a67-b7281f423069
    _updateRecordAfterPush resource://gre/modules/PushService.sys.mjs:845
[PushService.sys.mjs:813](resource://gre/modules/PushService.sys.mjs)
NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIFaviconService.replaceFaviconDataFromDataURL]
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:226
    loadFavicon resource:///modules/PlacesUIUtils.sys.mjs:729
    setIconFromLink resource:///actors/LinkHandlerParent.sys.mjs:139
    receiveMessage resource:///actors/LinkHandlerParent.sys.mjs:60
[LinkHandlerParent.sys.mjs:148:17](resource:///actors/LinkHandlerParent.sys.mjs)
SearchSuggestionController found an unexpected string value: Unexpected response, searchString does not match remote response [SearchSuggestionController.sys.mjs:634:17](resource://gre/modules/SearchSuggestionController.sys.mjs)
Alternate Service Mapping found: https://adservice.google.de:-1 to https://googleads.g.doubleclick.net:443
![grafik](https://github.com/SVG-Edit/svgedit/assets/135705834/e0ca10de-c8f2-4737-9fd3-1aacc23ef2f5)

**Reply by BS-eng2023** [reply]
actually I get the extention aktive when I am using a local server
![grafik](https://github.com/SVG-Edit/svgedit/assets/135705834/bc0d2802-a102-4d0d-a9ce-52122ef6088c)
